packages/components/src/internal/components/chart/ChartFieldAggregateOptions.tsx

```diff
@@ -0,0 +1,158 @@
+import { RadioGroupInput, RadioGroupOption } from '../forms/input/RadioGroupInput';
+
+import { BAR_CHART_AGGREGATE_NAME, BAR_CHART_ERROR_BAR_NAME } from './constants';
+import { ChartFieldInfo } from './models';
+import { LabelOverlay } from '../forms/LabelOverlay';
+import { SelectInput, SelectInputOption } from '../forms/input/SelectInput';
+
+const BAR_CHART_AGGREGATE_METHODS = [
+    { label: 'None', value: '' },
+    { label: 'Count (non-blank)', value: 'COUNT' },
+    { label: 'Sum', value: 'SUM' },
+    { label: 'Min', value: 'MIN' },
+    { label: 'Max', value: 'MAX' },
+    { label: 'Mean', value: 'MEAN' },
+    { label: 'Median', value: 'MEDIAN' },
+];
+const BAR_CHART_AGGREGATE_METHOD_TIP =
+    'The aggregate method that will be used to determine the bar height for a given x-axis category / dimension. Field values that are blank are not included in calculated aggregate values.';
+const BAR_CHART_ERROR_BAR_TIP =
+    "Show error bars on each bar representing Standard Deviation or Standard Error of the Mean. Only applicable for 'Mean' aggregate method.";
+
+const ERROR_BAR_TYPES = [
+    { value: undefined, label: 'None' },
+    { value: 'SD', label: 'Standard Deviation' },
+    { value: 'SEM', label: 'Standard Error of the Mean' },
+];
+
+interface OwnProps {
+    asOverlay?: boolean;
+    field: ChartFieldInfo;
+    fieldValues: Record<string, SelectInputOption>;
+    includeCount: boolean;
+    includeNone: boolean;
+    onErrorBarChange: (name: string, value: string) => void;
+    onSelectFieldChange: (name: string, value: string, selectedOption: SelectInputOption) => void;
+}
+
+export const ChartFieldAggregateOptions: FC<OwnProps> = memo(props => {
+    const {
+        field,
+        fieldValues,
+        onSelectFieldChange,
+        onErrorBarChange,
+        includeCount,
+        includeNone,
+        asOverlay = true,
+    } = props;
+    const fieldValue = fieldValues?.[field.name];
+    const aggregateValue = fieldValues?.[BAR_CHART_AGGREGATE_NAME]?.value;
+    const errorBarValue = fieldValues?.[BAR_CHART_ERROR_BAR_NAME]?.value;
+    const defaultAggregateValue = useMemo(() => (includeNone ? '' : 'SUM'), [includeNone]);
+    const errorBarRadioEnabled = useMemo(() => aggregateValue === 'MEAN', [aggregateValue]);
+
+    const aggregateOptions = useMemo(() => {
+        const options = BAR_CHART_AGGREGATE_METHODS.filter(option => {
+            if (option.value === 'COUNT' && !includeCount) {
+                return false;
+            }
+            if (option.value === '' && !includeNone) {
+                return false;
+            }
+            return true;
+        });
+
+        return options.map(
+            option =>
+                ({
+                    ...option,
+                    selected: aggregateValue === option.value,
+                }) as RadioGroupOption
+        );
+    }, [aggregateValue, includeCount, includeNone]);
+
+    const errorBarOptions = useMemo(() => {
```

Comment:
We shouldn't need to cast as RadioGroupOption, and we can move the hard coded `className` attribute to the constant values in ERROR_BAR_TYPES above.

```suggestion
        const errorBarOptions = useMemo(
        () =>
            ERROR_BAR_TYPES.map(option => ({
                ...option,
                disabled: !errorBarRadioEnabled,
                selected: errorBarValue === option.value,
            })),
        [errorBarRadioEnabled, errorBarValue]
    );
```

Reply:
the className wasn't needed / used, so removed